README.md

```diff
@@ -473,7 +473,8 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [You
 55th Annual Conf. on Information Sciences and Systems (CISS), 2021.
 - [Alzugaray, I., Ph.D. Thesis, 2022](#Alzugaray22PhD),  
 *Event-driven Feature Detection and Tracking for Visual SLAM*.
-
+- [Gao, et at., FPGA, 2022](#GAO22FPGA),  
```

Comment:
Hello. This work is not doing corner detection and tracking... so it should not be in this subcategory; it should be in the upper category "Feature Detection and Tracking". Could you move it, please?
Also, please check the key value works: Gao22fpga (case sensitive?).